common/src/main/java/org/apache/drill/common/scanner/ClassPathScanner.java

```diff
@@ -422,11 +423,16 @@ static ScanResult scan(Collection<URL> pathsToScan, Collection<String> packagePr
     }
   }
 
-  private static void verifyClassUnicity(List<AnnotatedClassDescriptor> annotatedClasses) {
+  private static void verifyClassUnicity(List<AnnotatedClassDescriptor> annotatedClasses, Collection<URL> pathsScanned) {
     Set<String> scanned = new HashSet<>();
     for (AnnotatedClassDescriptor annotated : annotatedClasses) {
       if (!scanned.add(annotated.getClassName())) {
-        throw new RuntimeException("BUG: " + annotated.getClassName() + " scanned twice");
+        throw UserException.functionError()
+            .message(
+                "function %s scanned twice in %s\nDo you have conflicting jars on the classpath?",
+                annotated.getClassName(), pathsScanned
```

Comment:
I did not know that, will have to remember it for future reference. In this case it might make sense to put the URLs on separate lines. The message is already multi-line and if these are going to be long paths it would make it easier to read the list and look for duplicated jars.